File: core_backend/core_backend/Models/GoogleMapsData.cs
using System.ComponentModel.DataAnnotations;

namespace core_backend.Models
{
    public class GoogleMapsData
    {
        [Key]
        public string Uuid { get; set; }
        public string GoogleId { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string Postcode { get; set; }
        public string Country { get; set; }
        public string Name { get; set; }
        public string UserId { get; set; }
    }
}

File: backend/models/google_maps_data.go
package models

type GoogleMapsData struct {
	Uuid     string `gorm:"primary_key" json:"uuid"`
	GoogleId string `json:"googleId"`
	Address  string `json:"address"`
	City     string `json:"city"`
	Postcode string `json:"postcode"`
	Country  string `json:"country"`
	Name     string `json:"name"`
	UserId   int64
}
